text/0017-message/0017-message.md

```diff
@@ -0,0 +1,169 @@
++ Feature name: `tangle-message`
++ Start date: 2020-07-28
++ RFC PR: [iotaledger/protocol-rfcs#0017](https://github.com/iotaledger/protocol-rfcs/pull/0017)
+
+# Summary
+
+A message is the object nodes gossip around in the network. It always references two other messages that are known as `parents`. It is stored as a vertex on the tangle data structure that the nodes maintain.
+
+The messages will contain payloads. Some of them will be core payloads that will be processed by all nodes as part of the core protocol. Some of them will be community payloads that will enable the building of new functionalities on top of the Tangle. Some payloads may have other nested payloads embedded inside.
```

Comment:
In the Payloads section below is says "A message _may_ contain a payload" but here it says "will", could you clarify if it is strictly required or not?